extensions/extensions.json

```diff
@@ -1,5 +1,6 @@
 [
   // This file supports comments
+  "StackOverflow/keysPlusV2",
```

Comment:
Havent looked at the extension. But you definitely dont get to put yourself at the front of the list. Maybe around some of the other keyboard ones. Like mobile keyboard.